docs/interfaces/ApplePayButtonInitializeOptions.md

```diff
@@ -12,7 +12,6 @@ DOM. When a customer clicks on it, it will trigger Apple sheet.
 ### Properties
 
 - [buyNowInitializeOptions](ApplePayButtonInitializeOptions.md#buynowinitializeoptions)
-- [isWebBrowserSupported](ApplePayButtonInitializeOptions.md#iswebbrowsersupported)
```

Comment:
You do not need to update these docs because it will be handled by default during bumping version of checkout-sdk